src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -77,7 +77,12 @@ internal bool Initialize(EngineController controller, WorkspaceModel workspace)
                 {
                     var node = workspace.Nodes.FirstOrDefault(n => n.GUID.Equals(nodeGuid));
                     if (node != null)
+                    {
                         node.ClearDirtyFlag();
+
+                        // Set IsEvaluating to true, so that node's preview bubble won't crash.
+                        node.IsEvaluating = true;
```

Comment:
Hi, @ke-yu , I tried to investigate it deeper. So, that's what I found. I'd like to draw it.
![image](https://cloud.githubusercontent.com/assets/8158404/13527255/818fa172-e216-11e5-87f4-9c27bbb36dc6.png)
![image](https://cloud.githubusercontent.com/assets/8158404/13527260/8a2f7ff0-e216-11e5-9299-875294b649e4.png)

You can easily see it in the next image.
![image](https://cloud.githubusercontent.com/assets/8158404/13527739/f55b38b6-e219-11e5-878e-869738bf3721.png)

So, another possible fix here is to reduce priority of DelegateBasedAsyncTask to `BelowNormal`.

Let me know, what you think.